src/hooks/useDropHandler.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+interface DragHandler<T> {
+    (e: React.DragEvent<T>): void;
+}
+
+function useDropHandler(
```

Comment:
Please use this in DropContainer

Reply:
Should not be related to this PR

Reply:
Yes, but please use

Reply:
Okay

Reply:
Both have slightly different feature set, so it can't be done at the moment